### DISCUSSION TITLE
be able to sell domains from paymenter

### DISCUSSION THREAD

**Discussion by yoygeimer** [Paymenter/Paymenter#352] [Ideas]
That you can sell domains from Paymenter as well as whcms is the only thing that Paymenter lacks to be 100% epic

**Comment by 65** [comment]
1000% this is required to have a full featured system. A good place to start would be Synergy Wholesale https://synergywholesale.com/wp-content/uploads/2024/05/Synergy-Wholesale-API-Documentation-v3-10.pdf or TPP https://support.tppwholesale.com.au/hc/en-gb/article_attachments/360009282597

**Comment by romanmiranda** [comment]
I discover today this repository looking for a WHMCS replacement. And yes, this should be an absolutely required feature. Wondering if there is already a plan for this or if is possible to start this feature from scratch?

**Comment by cubicloud-in** [comment]
its a must have feature for hosting companies and for paymenter since it is a whmcs alternative.

**Comment by apphostly** [comment]
I would say it would be enough to add support for domains (TLDs, pricing, whois, invoices, due dates/expire dates etc) and add extension support for Domains - then the community will chime in and create different registrar / reseller modules.
